### PR TITLE
Prepare for Release1.0.9.4

### PR DESCRIPTION
**Pull request by yophilav**
# 1.0.9.4 (2020-07-17)
### Bug Fixes
* Reestablish cloud connection after an exception is received ([212f1e0](https://github.com/Azure/iotedge/commit/212f1e0c7536b38b51a71828f4540f2853c2039d))
* Update [Linux ARM32 security vulnerable issue](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-June/005457.html) to OS version 2.1.19-bionic-arm32v7 ([51e707c](https://github.com/Azure/iotedge/commit/51e707c3d5e0cf630f91c9067c2c331753f2419c))

### Bug Fixes
* Get cloud connection if adding device with subscriptions ([7a7b550](https://github.com/Azure/iotedge/commit/7a7b550939d7ae7f7d4ba170ceb02023532481dc))
* Update [Linux ARM32 security vulnerable issue](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-June/005457.html) to OS version 2.1.19-bionic-arm32v7 ([51e707c](https://github.com/Azure/iotedge/commit/51e707c3d5e0cf630f91c9067c2c331753f2419c))

### Bug Fixes
* Edgelet unable to pull using certain passwords ([698bec5](https://github.com/Azure/iotedge/commit/698bec5b83d536e48d6b923eeeb2a518557973dd))

### Bug Fixes
* Update [Linux ARM32 security vulnerable issue](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-June/005457.html) to OS version 2.1.19-bionic-arm32v7 ([51e707c](https://github.com/Azure/iotedge/commit/51e707c3d5e0cf630f91c9067c2c331753f2419c))

### CODE REVIEW

File: edgelet/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.9.2
\ No newline at end of file
+1.0.9.3
```
Comment (darobs) [inline edgelet/version.txt:1]: Should this match edge agent/hub version?

Comment (yophilav) [reply edgelet/version.txt:1]: Nope, we didn't have an iotedged release for 1.0.9.3.  Now the edgelet is one behind.

Comment (varunpuranik) [reply edgelet/version.txt:1]: what have we done previously? Let's follow the same pattern.

Comment (varunpuranik) [reply edgelet/version.txt:1]: Looks like we did release edgelet for 1.0.9.3 - https://github.com/Azure/azure-iotedge/releases/tag/1.0.9.3

**Pull request merged**